scripts/core/ui/components/Form/SelectTagInput/index.tsx

```diff
@@ -124,7 +128,7 @@ export class SelectTagInput extends React.Component {
                     <LineInput noMargin={true} invalid={invalid} required={required}>
                         <Label text={label} />
 
-                        <div className="sd-line-input__input" readOnly={readOnly}>
```

Comment:
why are we removing `readOnly`?

Reply:
`readOnly` is an attribute for inputs and `div` is not an input. TypeScript flagged it an invalid property which it is.